ui/app/templates/components/mfa/mfa-setup-step-two.hbs

```diff
@@ -38,7 +38,13 @@
       </div>
     {{/if}}
     <div class="is-flex-start has-gap has-top-margin-l">
-      <button id="restart" type="button" class="button has-text-danger" {{on "click" this.restartSetup}} data-test-restart>
+      <button
+        id="restart"
+        type="button"
+        class="button is-danger-outlined"
```

Comment:
removed the `has-text-danger` class. There is now styling for the button type button.is-danger-outline. The syntax follows the pattern in the button.scss

Reply:
Additionally this was confusing because we have a helper with the same name "has-text-danger" which only styles the color and not the border.